File: tools/devctx/fixtures/formats/SampleService.swift
import Foundation
import UIKit

protocol UserServiceProtocol {
    func createUser(email: String) -> UUID
}

enum UserRole {
    case admin
    case member
}

struct UserDto {
    let id: UUID
    let email: String
}

actor SessionManager {
    func validate() -> Bool { return true }
}

class SampleService: UserServiceProtocol {
    func createUser(email: String) -> UUID {
        return UUID()
    }

    func deleteUser(id: UUID) {
        // delete logic
    }
}

func topLevelHelper() -> String {
    return "helper"
}

File: tools/devctx/fixtures/formats/SampleService.kt
package com.example.service

import java.util.UUID

interface UserService {
    fun createUser(email: String): UUID
}

enum class UserRole {
    ADMIN, MEMBER
}

data class UserDto(val id: UUID, val email: String)

object ServiceRegistry {
    fun register(service: UserService) {}
}

class SampleService : UserService {
    override fun createUser(email: String): UUID {
        return UUID.randomUUID()
    }

    fun deleteUser(id: UUID) {
        // delete logic
    }
}

fun topLevelHelper(): String {
    return "helper"
}
